test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -388,13 +388,13 @@ def test_mom_dyn_res_permissions(self):
 
         # Create script in a directory which has more open privileges
         # This should make loading of this file fail in all cases
-        dir_temp = self.du.mkdtemp(mode=0766, dir=home_dir)
+        dir_temp = self.du.mkdtemp(mode=0766, dir=home_dir, suffix=' tmp')
```

Comment:
there is a space in the suffix string, is that intentional?

Reply:
It is, my program failed when I ran it on windows because I had my script residing in "Program Files (x86)" which had spaces and the logic before this change interpreted script name as "Program". That is why I changed the test to add a space in the directory name.

Reply:
so you added the space because you wanted to test that your changes work even when the script is located in a folder which has spaces in its name? If that's the case then can you please add a comment here saying that you are adding the space for this reason?

Reply:
Sure, I'll add a comment